content/docs/tutorials/get-started/data-pipelines.md

```diff
@@ -137,59 +137,167 @@ stage as <abbr>dependencies</abbr> of another one, we can describe a sequence of
 commands that gets to a desired result. This is what we call a _data pipeline_
 or [_dependency graph_](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
 
-Let's create a second stage chained to the outputs of `prepare.dvc`, to perform
-feature extraction. And a third one for training a machine learning model, based
-on the features:
+Let's create a second stage chained to the outputs of `prepare`, to perform
+feature extraction:
 
 ```dvc
-$ dvc run -f featurize.dvc \
+$ dvc run -n featurize \
+          -p featurize.max_features,featurize.ngrams \
           -d src/featurization.py -d data/prepared \
           -o data/features \
-          python src/featurization.py data/prepared data/features
+          python src/featurization.py \
+                 data/prepared data/features
+```
+
+The `dvc.yaml` is updated automatically and should include two stages now.
+
+And should look like this:
```

Comment:
```suggestion
The `dvc.yaml` file is updated automatically and should include two stages
now.
```

Reply:
Removed the last sentence since the next section starts with "changes to the `dvc.yaml` should look like this" again.